src/cfnlint/customRules/Operators.py

```diff
@@ -2,17 +2,42 @@
 Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 SPDX-License-Identifier: MIT-0
 """
+# pylint: disable=W0622
+from cfnlint.rules import Match
 
 
-def equalsOp(rule, propertyList):
+def equalsOp(template, rule, propertyList):
     """ Process EQUALS operators """
+
     if not propertyList:
-        return 'EQUALS'  # Resource type not found
+        return Match(
+            1, 1,
+            1, 1,
+            template.filename, CustomRule('E9200'),
+            rule.operator, None)  # Resource type not found
+
     for prop in propertyList:
         actualValue = getProperty(prop, rule)
         if actualValue.strip() != str(rule.value).strip():
-            return 'Not Equal as ' + actualValue + ' does not equal ' + rule.value
-    return 'EQUALS'
+            path = prop['Path']
+            path = path + rule.prop.split('.')
+            linenumbers = template.get_location_yaml(template.template, path)
+            if linenumbers:
+                return Match(
+                    linenumbers[0] + 1, linenumbers[1] + 1,
+                    linenumbers[2] + 1, linenumbers[3] + 1,
+                    template.filename, CustomRule('E9201'),
+                    'Not Equal as ' + actualValue + ' does not equal ' + rule.value, None)
+            return Match(
+                1, 1,
+                1, 1,
```

Comment:
not sure if this is just temporary or not, but the `Resources` block won't always necessarily be at the beginning of a template if this code is for handling violations in code generated by the [Serverless transform](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/transform-aws-serverless.html) that aren't exactly present in the un-transformed template

---

Sidenote, you can see the Linter's view of transformed templates with:
[`cfn-lint --info template.yaml`](https://github.com/aws-cloudformation/cfn-python-lint/#serverless-application-model)

```yaml
Transform: AWS::Serverless-2016-10-31
Resources:
  Resource:
    Type: AWS::Serverless::SimpleTable
```
```json
{
  "Resources": {
    "Resource": {
      "Properties": {
        "AttributeDefinitions": [
          {
            "AttributeName": "id",
            "AttributeType": "S"
          }
        ],
        "BillingMode": "PAY_PER_REQUEST",
        "KeySchema": [
          {
            "AttributeName": "id",
            "KeyType": "HASH"
          }
        ]
      },
      "Type": "AWS::DynamoDB::Table"
    }
  }
}
```

Reply:
I'm presently using the existing code to locate the resource block. I believe the 1, 1, 1, 1 is the standard "fallback" used in the existing rule system for cases where the line number for some reason could not be found